### PR TITLE
test: adding test cases for participationrewards

### LINKED ISSUE #366: x/participationrewards tests
Newly exported functions in x/participationrewards should be tested:
- `AllocateHoldingsRewards()`
- `AllocateLockupRewards()`
- `AllocateValidatorSelectionRewards()`
- `CalcDistributionScores()`
- `CalcOverallScores()`
- `CalcUserValidatorSelectionAllocations()`

_Originally posted by @aljo242 in https://github.com/ingenuity-build/quicksilver/pull/357#pullrequestreview-1352690299_

### PR DESCRIPTION
**Pull request by muku314115**
## 1. Summary
Fixes #366
Note: CalcOverallScores never returns the error, instead it logs it and moves on, this behaviour has been changed in the PR to return the error in which case the getZoneScores function will stop on encountering any problem with overall score calculation

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Comment by aljo242** [comment]
structure looks good

**Comment by aljo242** [comment]
@muku314115 just lint failing

File: x/participationrewards/keeper/rewards_lockup_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package keeper_test
+
+import (
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/ingenuity-build/quicksilver/app"
+)
+
+func (s *KeeperTestSuite) TestAllocateLockupRewards() {
```
Comment (joe-bowman) [inline x/participationrewards/keeper/rewards_lockup_test.go:9]: should be `suite` rather than `s` to match the rest of the usages of `*KeeperSuiteTest`

**Review by joe-bowman — CHANGES_REQUESTED** [review]
looks good, but requires linting + gofumpt

File: x/participationrewards/keeper/rewards_lockup_test.go (hunk 1 above)
Comment (muku314115) [reply x/participationrewards/keeper/rewards_lockup_test.go]: done

**Review by joe-bowman — DISMISSED** [review]
lgtm

**Review by ajansari95 — APPROVED** [review]

**Review by aljo242 — APPROVED** [review]

**Pull request merged**